src/backend/gl/src/device.rs

```diff
@@ -846,13 +864,13 @@ impl d::Device<B> for Device {
             name
         };
 
-        let patch_size = match desc.input_assembler.primitive {
+        let patch_size = match desc_input_assembler.primitive {
             pso::Primitive::PatchList(size) => Some(size as _),
             _ => None,
         };
 
         let mut vertex_buffers = Vec::new();
-        for vb in &desc.vertex_buffers {
+        for vb in desc_vertex_buffers {
```

Comment:
nit: we should probably move that into the match arm, could be done as a follow-up